src/app/core/components/registration/registration.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="registration">
+	<span class="registration__name">
+		{{ loginService.authName$ | async }}
+	</span>
+
+	<mat-icon
+		color="primary"
+		*ngIf="!loginService.isUserAuthorised$">
+		person
```

Comment:
I suggest to use the next formatting: 
```
	<mat-icon
		color="primary"
		*ngIf="!loginService.isUserAuthorised$"
         >person</mat-icon>
```
or
```
	<mat-icon
		color="primary"
		*ngIf="!loginService.isUserAuthorised$"
         >
                person
         </mat-icon>
```
for better readability. In your case it seems that `person` is an attribute of `mat-icon` tag